app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -167,7 +170,7 @@ def form_response(destination:)
         extra: {
           destination: destination,
           skip_upload_step: mobile_device?,
-          flow_path: flow_session[:flow_path],
+          flow_path: idv_session.flow_path,
```

Comment:
Noting for myself and anyone else reviewing that this one is ok because idv_session.flow_path is always set earlier on this execution path.